templates/docs/document.html

```diff
@@ -0,0 +1,22 @@
+
+{% extends "docs/_base_docs.html" %}
+
+{% block title %}{{ document.title }} | Multipass documentation | {% endblock %}
```

Comment:
I'm not crazy about this solution - requiring creators of new templates to know and remember to add ` | ` to the end of their `title` block seems set up to fail.

If we can't come up with a default title in `_layouts/default.html` that works smoothly with the `title` block, we could use this slightly more complex syntax instead:

```
{% if self.title() %}
  <title>{{ self.title() }}</title>
{% else %}
  <title>Multipass - Orchestrates virtual Ubuntu instances</title>
{% endif %}
```

Reply:
Although, if you're going with a `Multipass documentation` suffix, it seems like we could probably let the `title` block override the entire thing:

``` html
<!--_layouts/default.html -->

<title>{% block title %}Multipass - Orchestrates virtual Ubuntu instances{% title %}</title>

<!-- docs/document.html -->
{% block title %}{{ document.title }} | Multipass documentation{% endblock %}
```